builder/builder.go

```diff
@@ -404,89 +404,109 @@ func (b *Builder) BuildUpdate(params UpdateParameters) error {
 	return nil
 }
 
-const imageTemplate = "release-image-config.json"
-const isterConfigDir = "/usr/share/defaults/ister"
+const migrationConfig = "release-image-config.json"
+const buildConfig = "release-image-config.yaml"
 
 // BuildImage will now proceed to build the full image with the previously
 // validated configuration.
-func (b *Builder) BuildImage(format string, template string) error {
+func (b *Builder) BuildImage(format string, configFile string) error {
 	// If the user did not pass in a format, default to builder.conf
 	if format == "" {
 		format = b.State.Mix.Format
 	}
 
-	// If the user did not pass in a template, use the default template
-	if template == "" {
-		template = imageTemplate
-		// If the default image template is not present in the mix workspace,
-		// copy it from the default ister config directory and update the bundle list based on mix bundles
-		templateFile := filepath.Join(b.Config.Builder.VersionPath, template)
-		if _, err := os.Stat(templateFile); os.IsNotExist(err) {
-			fmt.Printf("Warning: Image template %s not found\n", templateFile)
-			configFile := filepath.Join(isterConfigDir, template)
-			fmt.Printf("Copying image template from %s\n", configFile)
-			if err = b.copyImageTemplate(configFile, templateFile); err != nil {
+	// If the user did not pass in a configFile, use the default configFile
+	if configFile == "" {
+		configFile = filepath.Join(b.Config.Builder.VersionPath, buildConfig)
+
+		// First check if the previous ister.py configuration file exist,
+		// if so, then we need to migrate it to clr-installer YAML
+		migrationFile := filepath.Join(b.Config.Builder.VersionPath, migrationConfig)
```

Comment:
I believe it would be better to swap the order here and only checking for the ister config after we check for a clr-installer config. This way, if the user add a clr-installer config in an existing mix before running mixer, the config won't be overwritten by the conversion  of the old ister config.

Reply:
Pushed a new version which follows:
```
If YAML not exist:
    if JSON not exist:
        generate YAML
    else:
         migrate JSON to YAML

if JSON exists:
    rename JSON to JSON-EOL
```